packages/statusbar/src/defaults/memoryUsage.tsx

```diff
@@ -304,7 +304,7 @@ namespace Private {
   /**
    * The url endpoint for making requests to the server.
    */
-  const METRIC_URL = URLExt.join(SERVER_CONNECTION_SETTINGS.baseUrl, 'metrics');
+  const METRIC_URL = URLExt.join(SERVER_CONNECTION_SETTINGS.baseUrl, 'api/materials/v1');
```

Comment:
Should this be `api/metrics/v1`?

```suggestion
  const METRIC_URL = URLExt.join(SERVER_CONNECTION_SETTINGS.baseUrl, 'api/metrics/v1');
```